common/thorhelper/commonext.cpp

```diff
@@ -208,6 +208,8 @@ MODULE_INIT(INIT_PRIORITY_STANDARD)
     kindArray[TAKsmartdenormalizegroup] = "smartdenormalizegroup";
     kindArray[TAKselfdenormalize] = "selfdenormalize";
     kindArray[TAKselfdenormalizegroup] = "selfdenormalizegroup";
+    kindArray[TAKspillread] = "diskread" ;
+    kindArray[TAKspillwrite] = "diskwrite" ;
```

Comment:
Can these be "spilldiskread" and "spilldiskwrite", or do they have to have the same text labels?

Reply:
I'll change